api/web3/concordium.js

```diff
@@ -49,12 +51,45 @@ class Concordium {
       SchemaVersion.V2
     );
 
+    if (!account)
+    {
+      return returnValue.all_tokens;
+    }
+
     const data = returnValue.state.filter((accountInfo) => {
       return accountInfo[0].Account[0] === account;
     });
 
     return data[0][1].owned_tokens;
   }
+  async getNextNFT() {
+    return String(Object.keys(await this.listNFTs()).length + 1).padStart(
+        8,
+        "0"
+    );
+  }
+  async mintNFT(address, token) {
+    return await this.client.sendTransaction(
+        address,
+        AccountTransactionType.Update,
```

Comment:
Is there no need for the wallet private key to authorize the transaction?  Can I use any wallet and have it spend money? There is something missing here